CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Add Thread Pool Info context to event schema. ([#5153](https://github.com/getsentry/relay/pull/5153))
 - Add Unity Info context to event schema. ([#5155](https://github.com/getsentry/relay/pull/5155))
 - Generate `sentry.name` attributes for spans without names. ([#5143](https://github.com/getsentry/relay/pull/5143))
+- Add `retentions` to the project configuration. ([#5135](https://github.com/getsentry/relay/pull/5135))
```

Comment:
```suggestion
- Add `retentions` to the project configuration and use them for logs. ([#5135](https://github.com/getsentry/relay/pull/5135))
```

Reply:
nit: I would move this to the `Internal` section as it is not a user-visible behavior change.